src/librustdoc/html/layout.rs

```diff
@@ -113,7 +113,8 @@ crate fn render<T: Print, S: Print>(
     <section class=\"footer\"></section>\
     {after_content}\
     <div id=\"rustdoc-vars\" data-root-path=\"{root_path}\" data-current-crate=\"{krate}\" \
-       data-search-js=\"{root_path}search-index{suffix}.js\"></div>
+       data-search-index-js=\"{root_path}search-index{suffix}.js\" \
+       data-search-js=\"{root_path}search{suffix}.js\"></div>
```

Comment:
Oh actually just realized, this should be:

```suggestion
       data-search-js=\"{static_root_path}search{suffix}.js\"></div>
```